eng/pipelines/performance/templates/perf-build-jobs.yml

```diff
@@ -1,143 +1,164 @@
+      parameters:
+        jobTemplate: /eng/pipelines/common/global-build-job.yml
+        buildConfig: release
+        runtimeFlavor: mono
+        platforms:
+        - android_arm64
+        jobParameters:
+          buildArgs: -s mono+libs+host+packs -c $(_BuildConfig)
+          nameSuffix: AndroidMono
+          isOfficialBuild: false
+          postBuildSteps:
+            - template: /eng/pipelines/performance/templates/build-perf-sample-apps.yml
+              parameters:
+                rootFolder: '$(Build.SourcesDirectory)/artifacts/'
+                includeRootFolder: true
+                displayName: Android Mono Artifacts
+                artifactName: AndroidMonoarm64
+                archiveExtension: '.tar.gz'
+                archiveType: tar
+                tarCompression: gz
+                
+  - ${{ if containsValue(parameters.jobsAndPlatforms, 'mono') }}:
+    # build mono
+    - template: /eng/pipelines/common/platform-matrix.yml
+      parameters:
+        jobTemplate: /eng/pipelines/common/global-build-job.yml
+        buildConfig: release
+        runtimeFlavor: mono
+        platforms:
+        - linux_x64
+        jobParameters:
+          nameSuffix: mono
+          buildArgs: -s mono+clr.iltools+clr.hosts+libs+host+packs -c $(_BuildConfig)
+          isOfficialBuild: false
+          postBuildSteps:
+          - template: /eng/pipelines/common/upload-artifact-step.yml
             parameters:
-              rootFolder: '$(Build.SourcesDirectory)/artifacts/'
-              includeRootFolder: true
-              displayName: Android BDN App Artifacts
-              artifactName: PerfBDNAppArm
-              archiveExtension: '.tar.gz'
-              archiveType: tar
-              tarCompression: gz
-              perfBranch: ${{ parameters.perfBranch }}
+              rootFolder: $(Build.SourcesDirectory)/artifacts/bin
+              includeRootFolder: false
+              archiveType: $(archiveType)
+              archiveExtension: $(archiveExtension)
+              tarCompression: $(tarCompression)
+              artifactName: BuildArtifacts_$(osGroup)$(osSubgroup)_$(archType)_$(_BuildConfig)_mono
+              displayName: Build Assets
+              
+  - ${{ if containsValue(parameters.jobsAndPlatforms, 'monoPacks') }}:
+    # build mono runtime packs
+    - template: /eng/pipelines/common/platform-matrix.yml
+      parameters:
+        jobTemplate: /eng/pipelines/common/global-build-job.yml
+        buildConfig: release
+        runtimeFlavor: mono
+        platforms:
+        - android_arm64
+        jobParameters:
+          buildArgs: -s mono+libs+host+packs -c $(_BuildConfig)
+          nameSuffix: Mono_Packs
+          isOfficialBuild: false
+          postBuildSteps:
+            - template: /eng/pipelines/common/upload-intermediate-artifacts-step.yml
+              parameters:
+                name: MonoRuntimePacks
+                isOfficialBuild: false
```

Comment:
Not related to this PR as it is just a copy from the old code, but why do we build mono runtime packs with `android_arm64` platform?

Reply:
I also do not know. I assume they were getting used at some time. I suppose that is no longer the case?

Reply:
The mono runtime packs are used in the building of the perfBDN app. As to why android_arm64 over something like windows_arm64, I don't recall anything specific. I think it was to ensure it was labeled android in the pipeline view for easier picking out, but I don't think there is a requirement for android_arm64.

Reply:
Actually, I took another look, the other reason that android_arm64 was chosen is that influences the arch and os that the build.sh command passes in. So by choosing android_arm64, the android packages and what not are built targeting the android OS.